initfiles/componentfiles/thor/run_thor

```diff
@@ -35,7 +34,7 @@ echo $$ > $RUN_THOR_PID_NAME
 export SENTINEL="thor.sentinel"
 while [ 1 ]; do
     # update slaves file in case state of environment has been altered since last run
-    daliadmin $DALISERVER dfsgroup ${groupName} slaves
+    daliadmin server=$DALISERVER dfsgroup ${groupName} slaves
```

Comment:
Unless the dns name was <blah>= , which may not be legal, it should have not been caught by command processing.
But yes I agree, it's cleaner.